src/ngraph/op/experimental/layers/prior_box.hpp

```diff
@@ -68,6 +71,8 @@ namespace ngraph
                 virtual std::shared_ptr<Node>
                     copy_with_new_args(const NodeVector& new_args) const override;
 
+                static size_t number_of_priors(const PriorBoxAttrs& attrs);
```

Comment:
Do we really need static method?

What are use cases when we are using this method without PriorBox creation?